activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -172,6 +172,14 @@ def #{method_name}=(value)             # def includes_values=(value)
     # will work correctly:
     #
     #   User.includes(:posts).where(posts: { name: 'example' })
+    #
+    # Passing conditions to your included models will only include associations
+    # that pass the conditional in the result set. For example,
+    #
+    #   User.includes(:posts).where(posts: { name: 'example' })
+    #
+    # will only include posts with the name "example" even if a user has other
+    # posts that do not meet this condition.
```

Comment:
It seemed a little odd to me that we're duplicating the code example from just one line above.  I wondered if it was a typo, and I think a reader might wonder as well.  I took a stab at rewording this to reference the line above.  I also added a bit of emphasis to the point of confusion from #42743.  What do you think?

```suggestion
    # Conditions affect both sides of an association.  For example, the above 
    # code will return only users that have a post named "example", <em>and will 
    # only include posts named "example"</em>, even when a matching user has 
    # other additional posts.
```

Reply:
@jonathanhefner Thank you for the suggestion! I've updated the documentation as you requested.